src/utils/contentful/contentful.ts

```diff
@@ -1,15 +1,23 @@
 import { createClient, Environment } from 'contentful-management';
+import { createClient as createCDAClient } from 'contentful';
 import * as dotenv from 'dotenv';
 
 // load dotenv config
 dotenv.config();
 
-const client = createClient({
+export const CONTENTFUL_LIMIT = 25;
+
+export const cdaClient = createCDAClient({
```

Comment:
What's cda and what's cma? That's not super clear.

Reply:
I'm going to add more comments in these 2 sections. In short, CDA (Content Delivery API) is suitable for read-only operations and CMA (Content Management API) is more suitable for managing content.

From the docs in https://www.contentful.com/developers/docs/references/content-management-api/
```
Note: You can use the CMA to deliver and manage content, but you shouldn't use it to deliver large 
amounts of content and instead use the Content Delivery API. The structure of responses from the   
CMA differs from the CDA as GET responses retrieve the entirety of items (i.e. all localized and   
unpublished content).                                                                              
```

Reply:
It's going to looks something like this:
![Screenshot 2023-11-15 at 11 55 31 AM](https://github.com/thisdot/movies-api/assets/22064697/b6df3a36-fa29-423c-8e0a-f545a8da0da0)